src/db/migrations/20190115160348-create-updated-comment.js

```diff
@@ -9,6 +9,8 @@ module.exports = {
       },
       commentId: {
         type: Sequelize.INTEGER,
+        onDelete: 'CASCADE',
```

Comment:
I don't think its appriopriate to modify the migration file directly, instead create a new migration file for changing the column

Reply:
okay, will fix it